src/pages/Withdrawal/index.tsx

```diff
@@ -187,7 +187,7 @@ const Withdrawal: React.FC<WithdrawalProps> = ({
       const withdrawPset = wallet.sendTx(
         getRecipient(),
         customCoinSelector(dispatch),
-        changeAddress.confidentialAddress,
+        () => changeAddress.confidentialAddress,
```

Comment:
this is not a change related to bumping libraries, and woudl compromise any possibility to rollback or investigate regressions etc...